dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetDataTest.java

```diff
@@ -34,12 +51,18 @@ public void tearDown() {
     }
 
     @Test
-    public void testNewTokenNotExpired() {
-        System.out.println("newTokenNotExpired");
-        PasswordResetData instance = new PasswordResetData(null);
-        boolean expResult = false;
-        boolean result = instance.isExpired();
-        assertEquals(expResult, result);
+    public void testSetExpiredToSystemDefault() {
+
```

Comment:
It would be advisable to use BDD style of tests. So for example 


        // given
        DataverseTextMessageDto textMessageDto = mock(DataverseTextMessageDto.class);
        when(mapper.mapToNewTextMessage(1L)).thenReturn(textMessageDto);

        // when
        DataverseTextMessageDto retTextMessageDto = service.newTextMessage(1L);

        // then
        assertSame(textMessageDto, retTextMessageDto);`